DOL.WHD.Section14c.Web/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "dol-whd-14c",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "karma start --single-run",
+    "testd": "DEBUG=true karma start",
+    "start": "webpack-dev-server",
+    "clean": "rimraf ./dist/*",
+    "deploy": "npm run clean && webpack -p --config ./webpack.production.config.js && ncp index.html ./dist/index.html && ncp env.js ./dist/env.js"
+  },
+  "author": "",
+  "license": "ISC",
```

Comment:
PWS 5.5.1.a requires that deliverables be in the public domain.  Instead of ISC, 18F suggests `CC0-1.0` which is the SPDX code for the public domain commitment "license" that we use.